stack.yaml

```diff
@@ -0,0 +1,66 @@
+# This file was automatically generated by 'stack init'
```

Comment:
Yes. This is stack configuration file, kind of package description. It is only autogenerated in a sense that not hand-written from scratch, but created by the tool. It is not temporary, and not overwritten at build time.